.github/workflows/main.yml

```diff
@@ -1,4 +1,5 @@
-name: distkv_build
+name: distkv_ci
```

Comment:
Readme’s badge also need to be changed. 

Btw,you can replace underline to 20% in readme.